_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -262,6 +262,10 @@ typedef struct {
 
     /* WHL */
     { 0x3EA0, MFX_HW_CFL, MFX_GT2 },
+    { 0x3EA1, MFX_HW_CFL, MFX_GT1 },
+    { 0x3EA2, MFX_HW_CFL, MFX_GT3 },
+    { 0x3EA3, MFX_HW_CFL, MFX_GT2 },
+    { 0x3EA4, MFX_HW_CFL, MFX_GT1 },
```

Comment:
Part of the driver knows of these devices, but i915 uAPI part does not:
```
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3ea1 | grep RegisterDevice
media_driver/linux/gen9/ddi/media_sysinfo_g9.cpp:835:    RegisterDevice(0x3ea1, &cflGt1Info);
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3ea2 | grep RegisterDevice
media_driver/linux/gen9/ddi/media_sysinfo_g9.cpp:847:    RegisterDevice(0x3ea2, &cflGt3eInfo);
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3ea3 | grep RegisterDevice
media_driver/linux/gen9/ddi/media_sysinfo_g9.cpp:844:    RegisterDevice(0x3ea3, &cflGt2Info);
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3ea4 | grep RegisterDevice
media_driver/linux/gen9/ddi/media_sysinfo_g9.cpp:838:    RegisterDevice(0x3ea4, &cflGt1Info);
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3EA1 | grep RegisterDevice
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3EA2 | grep RegisterDevice
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3EA3 | grep RegisterDevice
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3EA4 | grep RegisterDevice
[dvrogozh@dvrscl media-driver]$
```
I.e. there are no matches in https://github.com/intel/media-driver/blob/master/media_driver/linux/common/os/i915/include/intel_chipset.h. I am not sure whether this can lead to driver side issues.